cmake.deps/cmake/UnibiliumCMakeLists.txt

```diff
@@ -6,16 +6,7 @@ add_library(unibilium ${SRC_FILES})
 set_target_properties(unibilium PROPERTIES PUBLIC_HEADER ${PROJECT_SOURCE_DIR}/unibilium.h
   VERSION "${VERSION_MAJOR}.${VERSION_MINOR}")
 
-if(NOT WIN32)
-  execute_process(COMMAND "shell ncursesw6-config --terminfo-dirs 2>/dev/null || \
-      ncurses6-config  --terminfo-dirs 2>/dev/null || \
-      ncursesw5-config --terminfo-dirs 2>/dev/null || \
-      ncurses5-config  --terminfo-dirs 2>/dev/null || \
-      echo '/etc/terminfo:/lib/terminfo:/usr/share/terminfo:/usr/lib/terminfo:/usr/local/share/terminfo:/usr/local/lib/terminfo'"
-    OUTPUT_VARIABLE TERMINFO_DIRS)
-else()
-  set(TERMINFO_DIRS "\"\"")
-endif()
+set(TERMINFO_DIRS "\"\"")
```

Comment:
> Remove non-windows code from dependencies if they're only used in Windows.

Aren't these "bundled" deps which could be used on any system?

Reply:
They could be, but they aren't. On `BuildUnibilium.cmake` we see that `UnibiliumCMakeLists.txt` is only used on Windows. But still, maybe it's safer to leave these files untouched in case we start use them for non-windows systems in the future?

Reply:
That seems like a confusing bit of knowledge to implicitly encode in our bundled builds. (Side note: does it make sense for us to add CMakeLists.txt to https://github.com/neovim/unibilium ? )